sparse_strips/vello_api/src/lib.rs

```diff
@@ -1,9 +1,27 @@
 // Copyright 2025 the Vello Authors
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 
-//! This crate defines the public API types, providing a stable interface for CPU and hybrid
-//! CPU/GPU rendering implementations. It provides common interfaces and data structures used
-//! across different implementations
+//! This crate defines the public API types used by both Vello CPU and Vello Hybrid.
+//!
+//! ## Usage
+//!
+//! This crate should not be used on its own, and you should instead use one of the renderers which use it.
+//! At the moment, only [Vello CPU](crates.io/crates/vello_cpu) is published, and you probably want to use that.
+//!
+//! We also develop [Vello](crates.io/crates/vello), which makes use of the GPU for 2D rendering and has higher performance than Vello CPU.
+//! Vello CPU is being developed as part of work to address shortcomings in Vello.
+//! Vello does not use this crate.
+//!
+//! ## Features
+//!
+//! - Shared API types for Vello's rendering pipeline.
+// - Interfaces for render contexts and rendering options.
+// - Designed for compatibility across CPU and GPU implementations.
+//!
+//! ## Usage
+//!
+//! This crate is intended to be used by other Vello components.
+//and external consumers needing a stable API.
```

Comment:
Remove?

Reply:
Sure? I was trying to preserve the text which was here before, because I don't know what the plan is with Vello API. My preference would be to be rid of it entirely, and move everything it contains into Vello Common.

Reply:
Makes sense.

> move everything it contains into Vello Common

I agree.